utils/beamSpotMerge.py

```diff
@@ -8,16 +8,18 @@
 sys.path.append('..')
 from objects.BeamSpot import BeamSpot
 
-def cleanAndSort(fullList, cleanBadFits = True):
+def cleanAndSort(fullList, cleanBadFits = True, iov = False):
     '''
     Sorts the lumi:BS dictionary and cleans it up
     from the not properly converged fits.
     '''
     # clean from badly converged
     cleaned =  {k:v for k, v in fullList.items() if v.Type > 0 and cleanBadFits}
-
+    if not iov:
```

Comment:
can you move this line below the next line `# sort by LS` ?

Reply:
Done ;)